test/enginetest.c

```diff
@@ -12,12 +12,13 @@
 #include <stdlib.h>
 #include <openssl/e_os2.h>
 
+# include "testutil.h"
```

Comment:
I'm not sure it's worth another commit to remove the extra indentation in the copy/paste'd line.